types/mitsobox/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for mitsobox 0.1
+// Project: https://github.com/jimfilippou/mitsobox
+// Definitions by: MrClemds <https://github.com/MrClemds>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
```

Comment:
Why does this require TypeScript 3.7?

Reply:
I thought it was necessary to indicate this information and so I added the one I use.